Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

```diff
@@ -142,6 +142,39 @@ Follow these steps to schedule an instance.
 
   Besides the user interface, you can automatically schedule instances using the instances API with a [POST or PUT](https://www.ctl.io/api-docs/cam/#application-lifecycle-management-instances-api) request.
 
+### Protecting Instance Shutdown/Termination
+
+Avoid accidental operations on particular instances. *Instance Protection Flag* allows to protect an instance for manual **shutdown** or **termination**. Instance still being affected by *scheduled termination or shutdown*. 
+
+*Instance Protection Flag* forbid operators to run shutdown/terminate operations on API/UI level, also affecting at **provider level** if implements this feature. 
```

Comment:
```suggestion
*Instance Protection Flag* prevents the user to perform shutdown/terminate operations through the user interface or an API call. It can also be enabled at **provider level**, affecting all instances deployed on it. 
```